app/assets/stylesheets/cart.css.scss

```diff
@@ -9,6 +11,12 @@
   font-size: 42px;
 }
 
+#cart-title{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space